lib/chef/provider/remote_file/cache_control_data.rb

```diff
@@ -145,18 +145,51 @@ def load_data
         end
 
         def load_json_data
-          Chef::FileCache.load("remote_file/#{sanitized_cache_file_basename}")
+          path = sanitized_cache_file_path(sanitized_cache_file_basename)
```

Comment:
Maybe instead of doing this as a one off, we should put some kind of version number in the metadata this time so we  can do this more cleanly next time?